server/grpc_service.go

```diff
@@ -1787,7 +1788,12 @@ func (s *GrpcServer) getGlobalTSOFromTSOServer(ctx context.Context) (pdpb.Timest
 	})
 	ts, err := forwardStream.Recv()
 	if err != nil {
-		log.Error("get global tso from tso server failed", zap.Error(err))
+		log.Error("get global tso from tso service primary addr failed", zap.Error(err), zap.String("tso", forwardedHost))
+		if strings.Contains(err.Error(), "transport is closing") {
```

Comment:
What if it is another kind of error?

Reply:
Update the client for any errors encountered? I'm not sure if this will keep updating repeatedly

Reply:
Can you add more descriptions about why this issue happens?

Reply:
Then, how to ensure that this situation does not happen again? Maybe we need to first know how many kinds of errors it might return.

Reply:
maybe it is better to judge by error code https://github.com/grpc/grpc-go/blob/16651f60ddc5b1adb810fca8b1a732d57fecb270/codes/codes.go#L180

"transport is closing" is defined in https://github.com/grpc/grpc-go/blob/3151e834fa2536844f801bb132c684fbb10364fd/internal/transport/transport.go#L782

Reply:
And I will add retry with new client for this error